### PR TITLE
item mod fixes

### PR DESCRIPTION
**Pull request by Ninjistix**
various fixes, 1 correction

### CODE REVIEW

File: sql/item_mods.sql [hunk 1]
```diff
@@ -25531,7 +25563,15 @@ INSERT INTO `item_mods` VALUES (22076,25,20);     -- Accuracy+20
 INSERT INTO `item_mods` VALUES (22076,346,5);     -- Avatar Perpetuation-5
 INSERT INTO `item_mods` VALUES (22077,23,25);     -- was_+1 Attack+25
 INSERT INTO `item_mods` VALUES (22077,25,25);     -- Accuracy+25
-INSERT INTO `item_mods` VALUES (22077,346,7);     -- Avatar Perpetuation-7
+INSERT INTO `item_mods` VALUES (22077,346,-7);     -- Avatar Perpetuation-7
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: https://github.com/DarkstarProject/darkstar/blob/11cf09da0b81f199ee9abf4fc82d9bb28d20cad9/src/map/status_effect_container.cpp#L1425
looks like that expects positive values?

File: sql/item_mods.sql [hunk 2]
```diff
@@ -25589,6 +25629,9 @@ INSERT INTO `item_mods` VALUES (22281,23,23);     -- knobkierrie Attack+23
 INSERT INTO `item_mods` VALUES (22281,841,6);     -- Weapon Skill Damage+6%
 INSERT INTO `item_mods` VALUES (22115,355,200); -- Yoichinoyumi iLvL 119 - Namas Arrow
 INSERT INTO `item_mods` VALUES (22116,355,202); -- Gandiva iLvL 119 - Jishnu's Raidance
+-- INSERT INTO `item_mods` VALUES (22118,Minuet: Ranged Attack,35); -- Venery bows Minuet: Ranged Attack+35
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: that looks like a latent to me

File: sql/item_mods.sql [hunk 3]
```diff
@@ -29847,6 +29907,22 @@ INSERT INTO `item_mods` VALUES (25668,384,30);    -- Haste +30
 INSERT INTO `item_mods` VALUES (25669,1,2);       -- crab_cap_+1 DEF:2
 INSERT INTO `item_mods` VALUES (25677,1,1);       -- Arthro's Cap DEF:+1
 INSERT INTO `item_mods` VALUES (25678,1,2);       -- Arthro's Cap +1 DEF:+2
+INSERT INTO `item_mods` VALUES (25704,1,133); -- DEF,133
+INSERT INTO `item_mods` VALUES (25704,2,63); -- HP,63
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: whats with the commas

File: sql/item_mods.sql [hunk 4]
```diff
@@ -38457,10 +38556,17 @@ INSERT INTO `item_mods` VALUES (27507,25,5);
...
+INSERT INTO `item_mods` VALUES (27516,311,10); -- Magic Damage +10
 INSERT INTO `item_mods` VALUES (27524,2,25);      -- Nodens Gorget: HP +25
 INSERT INTO `item_mods` VALUES (27524,5,25);      -- MP +25
 INSERT INTO `item_mods` VALUES (27524,374,5);     -- Cure potency +5%
 INSERT INTO `item_mods` VALUES (27524,539,30);    -- Enhances Stoneskin effect +30
+INSERT INTO `item_mods` VALUES (27531,14,3); -- Darkside earring CHR +3
+INSERT INTO `item_mods` VALUES (27531,119,5); -- Singing skill +5
+INSERT INTO `item_mods` VALUES (27531,121,5); -- Wind instrument skill +5
+INSERT INTO `item_mods` VALUES (27531,120,3); -- String instrument skill +5
+-- INSERT INTO `item_mods` VALUES (27531,Minne: Physical damage taken,-3); -- Minne: Physical damage taken -3% [unassign value]
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: latent prolly

File: sql/item_mods.sql [hunk 5]
```diff
@@ -42609,6 +42746,22 @@ INSERT INTO `item_mods` VALUES (28032,14,9);
 INSERT INTO `item_mods` VALUES (28032,31,20);
 INSERT INTO `item_mods` VALUES (28032,68,5);
 INSERT INTO `item_mods` VALUES (28032,384,30);
+INSERT INTO `item_mods` VALUES (28034,1,84); -- Dynasty mitts 28034 DEF:84
```
Comment (TeoTwawki) [inline sql/item_mods.sql]: the department of redundancy department wants us to know about item ids 😄

File: sql/item_mods.sql (hunk 4 above)
Comment (Ninjistix) [reply sql/item_mods.sql]: so would it be correct to assume legion/besieged and such other effects as latents too?

**Comment by Ninjistix** [comment]
removed the dup item number, changed the comma's i missed back to + and removed the possible latent's

File: sql/item_mods.sql [hunk 6]
```diff
@@ -25378,6 +25393,10 @@ INSERT INTO `item_mods` VALUES (21364,30,4);
 INSERT INTO `item_mods` VALUES (21366,8,3);
 INSERT INTO `item_mods` VALUES (21366,12,5);
 INSERT INTO `item_mods` VALUES (21366,23,10);
+INSERT INTO `item_mods` VALUES (21367,11,3); -- Tengu-no-Hane AGI+3
+INSERT INTO `item_mods` VALUES (21367,289,3); -- "Subtle Blow"+3
+-- INSERT INTO `item_mods` VALUES (21367,Daytime: Accuracy,15); -- Daytime: Accuracy+15
```
Comment (Ninjistix) [inline sql/item_mods.sql]: @TeoTwawki would this be latent too? the daytime? or item script?

Comment (TeoTwawki) [reply sql/item_mods.sql]: yeah and that one exists pretty sure

Comment (TeoTwawki) [reply sql/item_mods.sql]: yup!
LATENT_TIME_OF_DAY = 26, //PARAM: 0: DAYTIME 1: NIGHTTIME 2: DUSK-DAWN
can go in latent sql table

Comment (Ninjistix) [reply sql/item_mods.sql]: okay removed the 2 lines

**Pull request merged**